src/d3a/d3a_core/sim_results/endpoint_buffer.py

```diff
@@ -91,7 +107,9 @@ def generate_json_report(self):
             "tree_summary": self.tree_summary,
             "status": self.status,
             "device_statistics": self.device_statistics.device_stats_time_str,
-            "energy_trade_profile": self.energy_trade_profile
+            "energy_trade_profile": self.energy_trade_profile,
+            "last_unmatched_loads": self.last_unmatched_loads,
+            "last_energy_trade_profile": self.last_energy_trade_profile
```

Comment:
Why are you adding these to the local export? Is this a remnant of testing?

Reply:
Indeed remnant of testing, removing these. Thanks!